### ISSUE TITLE
Specify transformations in config convenience functions

### ISSUE THREAD

**Issue by jdeschamps** [CAREamics/careamics#231]
Currently, users can only interact with the transformations by enabling/disabling them in the convenience functions:

```python
config = create_n2v_configuration(
    experiment_name="convallaria_structn2v",
    data_type="array",
    axes="YX",
    patch_size=(64, 64),
    batch_size=16,
    num_epochs=50,
    use_augmentations=False,
)
```

To have a single x-axis flip augmentation (rather than x and y-flips, and 90 degrees rotations), users would have to do the following:

```python
from careamics.config.transformations import XYFlip

config.data_config.transforms.insert(
    0,
    XYFlip(flip_x=True, flip_y=False),
)
```

This is not very convenient and would be better to be able do the following:

```python
from careamics.config.transformations import XYFlip

config = create_n2v_configuration(
    experiment_name="convallaria_structn2v",
    data_type="array",
    axes="YX",
    patch_size=(64, 64),
    batch_size=16,
    num_epochs=50,
    
    # new parameters
    augmentations = [XYFlip(flip_x=True, flip_y=False)]

    # default
    use_augmentations=True,
)
```
